src/components/chip-set/chip-set.tsx

```diff
@@ -59,6 +59,7 @@ const INPUT_FIELD_TABINDEX = 1;
  * @exampleComponent limel-example-chip-set-input-type-text
  * @exampleComponent limel-example-chip-set-input-type-search
  * @exampleComponent limel-example-chip-icon-color
+ * @exampleComponent limel-example-chip-set-image
```

Comment:
in a followup PR, we can use the new `Img` interface and enable images in `limel-list` as well, which will enable us to add pictures in the list of suggestions of `limel-picker`